Formula/tigmint.rb

```diff
@@ -17,23 +17,27 @@ class Tigmint < Formula
   depends_on "minimap2"
   depends_on "numpy"
   depends_on "python@3.9"
+  depends_on "samtools"
 
   def install
     ENV.prepend_path "PATH", libexec/"bin"
     xy = Language::Python.major_minor_version "python3"
     ENV.prepend_path "PYTHONPATH", libexec/"lib/python#{xy}/site-packages"
     inreplace "bin/tigmint-cut", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
-    inreplace "bin/tigmint-molecule", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
-    inreplace "bin/long-to-linked", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
+    inreplace "bin/tigmint_molecule.py", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
+    inreplace "bin/tigmint_molecule_paf.py", "/usr/bin/env python3", Formula["python@3.9"].bin/"python3.9"
```

Comment:
I'd suggest `tigmint-molecule-paf` for consistency with other executable names in the package.